app/views/layouts/_nav.html.erb

```diff
@@ -1,16 +1,28 @@
 <nav class="text-center">
-  <a href="/#participate"><%=t "Contributors" %></a>
-
+  <% if current_user %>
+    <a href="/#participate"><%=t "Contributors" %></a>
+  <% else %>
+    <a href="/<%= "?locale=#{I18n.locale}" if I18n.locale != :en %>#participate"><%=t "Contributors" %></a>
```

Comment:
```suggestion
    <a href="/<%= "?locale=#{I18n.locale}" if I18n.locale != :en %>#participate"><%=t "Contributors" %></a>
```

I'd like to see the i18n logic and `current_user` logic pulled into a (new) helper as it's repeated three times.

Thanks!

Reply:
@BaseMax nudge on this when you get a chance!

Reply:
Hi Mike, Sure I will test this solution as soon as possible. Thanks @MikeMcQuaid